baseclasses/solvers/BaseSolver.py

```diff
@@ -224,18 +223,18 @@ def printModifiedOptions(self):
         modifiedOptions = self.getModifiedOptions()
         self.pp(modifiedOptions)
 
-    def pp(self, obj, flush=False):
+    def pp(self, obj, flush=True):
         """
         This method prints ``obj`` (via pprint) on the root proc of ``self.comm`` if it exists.
         Otherwise it will just print ``obj``.
 
         Parameters
         ----------
         obj : object
-            any Python object to be printed
+            Any Python object to be printed
+        flush : bool
+            If defined, the stream will be flushed.
```

Comment:
I'm not sure about the wording here. `flush` can be defined to be False.

Reply:
Fixed